### PR TITLE
content/en/docs: v17 --builtinbackup-file-read-buffer-size and --builtinbackup-file-write-buffer-size

### PR DESCRIPTION
**Pull request by maxenglander**
Accompanies https://github.com/vitessio/vitess/pull/12073

### CODE REVIEW

**Comment by rsajwani** [comment]
You might need to change it to v17

**Comment by maxenglander** [comment]
@rsajwani looks like there isn't a v17 folder or branch yet. Is there a process I should follow, or just `cp -R 16.0 17.0` folder and find-replace?

File: content/en/docs/17.0/reference/programs/vtctld.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+| --backup_engine_implementation | string | Specifies which implementation to use for creating new backups (builtin or xtrabackup). Restores will always be done with whichever engine created a given backup. (default "builtin") |
+| --backup_storage_block_size | int | if backup_storage_compress is true, backup_storage_block_size sets the byte size for each block while compressing (default is 250000). (default 250000) |
+| --backup_storage_compress | boolean | if set, the backup files will be compressed (default is true). Set to false for instance if a backup_storage_hook is specified and it compresses the data. (default true) |
+| --backup_storage_implementation | string | which implementation to use for the backup storage feature |
+| --backup_storage_number_blocks | int | if backup_storage_compress is true, backup_storage_number_blocks sets the number of blocks that can be processed, at once, before the writer blocks, during compression (default is 2). It should be equal to the number of CPUs available for compression (default 2) |
+| --builtinbackup-file-read-buffer-size | uint | read files from disk in blocks of this many bytes. Golang defaults are used when set to 0. |
```
Comment (maxenglander) [inline content/en/docs/17.0/reference/programs/vtctld.md]: This line and the next are new

File: content/en/docs/17.0/reference/programs/vtctld.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
+---
```
Comment (maxenglander) [inline content/en/docs/17.0/reference/programs/vtctld.md]: Except where noted, the contents of this file are just copy-pasted from `content/en/docs/16.0/reference/programs/vtctld.md`

File: content/en/docs/17.0/reference/programs/vttablet.md [hunk 3]
```diff
@@ -0,0 +1,447 @@
+---
```
Comment (maxenglander) [inline content/en/docs/17.0/reference/programs/vttablet.md]: Except where noted, the contents of this file are just copy-pasted from `content/en/docs/16.0/reference/programs/vttablet.md`

File: content/en/docs/17.0/reference/programs/vttablet.md [hunk 4]
```diff
@@ -0,0 +1,447 @@
...
+| --binlog_user                                                    | string        | PITR restore parameter: username of binlog server.                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| --builtinbackup-file-read-buffer-size                            | uint          | read files from disk in blocks of this many bytes. Golang defaults are used when set to 0.                                                                                                                                                                                                                                                                                                                                                                                                                 |
```
Comment (maxenglander) [inline content/en/docs/17.0/reference/programs/vttablet.md]: This line and the next are new.

**Review by deepthi — APPROVED** [review]

**Review by rsajwani — COMMENTED** [review]
LGTM

**Comment by rsajwani** [comment]
> @rsajwani looks like there isn't a v17 folder or branch yet. Is there a process I should follow, or just `cp -R 16.0 17.0` folder and find-replace?

> My apology for not getting back on this earlier. Looks like this is taken care of now.

**Pull request merged**